Added the pinged IP address to alfred notification output

blizzrdof77 opened the pull request.

blizzrdof77 committed:
Added the pinged IP address to the notification center text ouptut

blizzrdof77 committed:
Fixed grammar in ping notification

Changed "a" to "at" in the notification output text.

fniephaus commented on src/info.plist:
```diff
@@ -216,7 +216,7 @@ echo "DNS cache flushed"</string>
 				<integer>63</integer>
 				<key>script</key>
-				<string>ping -c 1 -t 6 {query} &amp;&gt; /dev/null &amp;&amp; echo "{query} is online." || echo "{query} is offline."</string>
+				<string>ping -c 1 -t 6 {query} &amp;&gt; /dev/null &amp;&amp; echo "{query} is online at $(command ping -c1 {query} | sed -nE 's/^PING[^(]+\(([^)]+)\).*/\1/p')." || echo "{query} is offline."</string>
```
Why is `command` needed in this line? Wouldn't it just work without it too?

blizzrdof77 replied:
It is a very rare use case, but some shell integrations (i.e _ZSH plugins_) override the default command for `ping` with a function. In my case, `ping` is actually a function that accepts and parses URLs in addition to domains (e.g. `ping https://github.com`). However, the fallacy would actually be on the plugin in this case, and it is unlikely that this would be loaded into the bash shell within Alfred anyways, so I'd say it's safe to take out!

fniephaus reviewed (changes requested):
Thanks for the contribution! I have one minor question, but otherwise happy to integrate it.

blizzrdof77 committed:
Remove 'command' from ping for IP in notification

Removing 'command' from ping for IP address in notification as requested by @fniephaus.

It was initially added because in rare cases, some shell integrations (i.e ZSH plugins) override the default command for ping with a function. In my case, ping is actually a function that accepts and parses URLs in addition to domains (e.g. ping https://github.com). 

However, the fallacy would actually be on the plugin in this case, and it is unlikely that this would be loaded into the bash shell within Alfred anyways, so we're removing it.

blizzrdof77 reviewed (commented):
@fniephaus - Updated!

fniephaus reviewed (approved):
LGTM...Thanks!
